### PR TITLE
fix assertions field type to Vec

### PR DESCRIPTION
**Pull request by zTgx**
as the title.

### CODE REVIEW

File: tee-worker/litentry/core/credentials/src/lib.rs [hunk 1]
```diff
@@ -358,8 +362,7 @@ impl Credential {
 				let timestamp = AssertionLogic::new_item("$timestamp", Op::GreaterEq, &from_date);
 
 				let assertion = AssertionLogic::new_and().add_item(lit_amounts).add_item(timestamp);
-
-				credential.credential_subject.assertions = assertion;
+				credential.credential_subject.set_assertion(assertion);
```
Comment (Kailai-Wang) [inline tee-worker/litentry/core/credentials/src/lib.rs]: I think `push` is better? Maybe we don't need `set_assertion` as `push` is native for `Vec`

**Review by Kailai-Wang — COMMENTED** [review]
Thanks, a small thing:

File: tee-worker/litentry/core/credentials/src/lib.rs (hunk 1 above)
Comment (zTgx) [reply tee-worker/litentry/core/credentials/src/lib.rs]: Yes, it's redundant. fields `assertions` and `values`  is a one-to-one relationship. will refactor `set_assertions` and `set_values` both.

**Review by jingleizhang — APPROVED** [review]
Looks good!

**Pull request merged**